WooCommerce/Classes/ViewRelated/Dashboard/Settings/CardReadersV2/SetUpTapToPayTryPaymentPromptViewController.swift

```diff
@@ -116,7 +122,7 @@ struct SetUpTapToPayPaymentPromptView: View {
         SimplePaymentsSummary(
             dismiss: {
                 viewModel.summaryActive = false
-                paymentFlowFinished = true
+                viewModel.paymentFlowFinished = true
```

Comment:
I mean it's a judgement call, and I'm sure I've done it the other way a bunch of times... but I don't think that adds to readability, since there's no extra logic there.

If I were to change it, I'd probably go with:

## View Model
```
func onTrialPaymentFlowFinished() {
    summaryActive = false
    paymentFlowFinished = true
}
```

## View
```
SimplePaymentsSummary(
    dismiss: viewModel.onTrialPaymentFlowFinished,
    rootViewController: rootViewController?.navigationController,
    [...]
```

Reply:
Looking further, `summaryActive` is set when the `summaryViewModel` is set, so nilling that out as well adds a bit of consistency.

Reply:
Actually, no, nilling that (at the wrong time) breaks the refund. Safer not to